.tina/collections/company.tsx

```diff
@@ -21,6 +22,8 @@ export const companySchema: Collection = {
   fields: [
     // @ts-ignore
     seoSchema,
+    // @ts-ignore
+    testimonialRowSchema,
```

Comment:
Do we still need this? It doesn't look like it's being used on the testimonials page, and its also not ideal to have this as a global field across all company pages.